cloverage/project.clj

```diff
@@ -28,9 +28,14 @@
                    :injections [(require 'pjstadig.humane-test-output)
                                 (pjstadig.humane-test-output/activate!)]
                    :plugins [[lein-cljfmt "0.6.4"]
-                             [jonase/eastwood "0.3.6"]
+                             [jonase/eastwood "0.8.1"]
                              [lein-kibit "0.1.7"]]
-                   :eastwood {:exclude-linters [:no-ns-form-found]}
+                   :eastwood {:exclude-linters [:no-ns-form-found]
+                              :ignored-faults {:unused-ret-vals {cloverage.sample.exercise-instrumentation true}
```

Comment:
If you ask me, the absolute best fix IMO would be to move `cloverage.sample.exercise-instrumentation` to dev-resources or test-resources. In the end `cloverage.sample.exercise-instrumentation` is not a ns that end users should be able to `require`.

And as a side-effect, by virtue of residing under `*resources`, the file wouldn't be linted at all by Eastwood, which I believe would be the right thing (strictly speaking the file is lintable, but it's neither intended as an actual `src` file or `test` file. So why bother?)

Didn't want to create an excessively opinionated PR though!

Reply:
That makes a lot of sense! I'd support that PR if you're up for writing it.